jujugui/static/gui/src/app/assets/javascripts/jujulib/charmstore.js

```diff
@@ -160,15 +160,15 @@ var module = module;
           charmMeta = meta['charm-metadata'],
           charmConfig = meta['charm-config'],
           bundleMeta = meta['bundle-metadata'],
-          bzrOwner = extraInfo['bzr-owner'];
+          owner = meta.owner && meta.owner.User;
```

Comment:
Owner was updated as a side effect of this branch which should fix the issues we were seeing where promulgated charms for certain users wouldn't properly link correctly.